source/CMakeLists.txt

```diff
@@ -47,5 +47,4 @@ message("nutrition-solver include dirs=${INCLUDE_DIRS}")
 #######################################################
 # LOCAL_PATH is unfortunately hard-coded for now
 set(RESOURCES_PATH "${MODERN_NUTRITION_ROOT}/resources/")
-target_compile_definitions(nutrition-solver PRIVATE -DRESOURCES_PATH=\"${RESOURCES_PATH}\")
-
+target_compile_definitions(nutrition-solver PUBLIC -DRESOURCES_PATH=\"${RESOURCES_PATH}\")
```

Comment:
Deixe como público apenas se for usar em arquivos `.h`. O melhor para esse tipo de coisa é normalmente ficar escondido no `.cpp` pois é apenas uma opção para debugar o código e não ficar exposto como parte da API para quem usar como dependência.